Corona-Warn-App/build.gradle

```diff
@@ -314,7 +314,7 @@ dependencies {
     // ANDROID STANDARD
     def nav_version = "2.3.3"
     implementation 'androidx.appcompat:appcompat:1.2.0'
-    implementation 'androidx.core:core-ktx:1.3.2'
+    implementation 'androidx.core:core-ktx:1.5.0'
```

Comment:
To use `WindowInsetsControllerCompat` which has a nice system UI visibility handling